Inventory.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Text;
+
```

Comment:
NIT : I noticed that `namespace` is not used in all of the files.
Adding a namespace to the files will help improve code organization .